mat-rest-commons/pom.xml

```diff
@@ -44,6 +43,12 @@
             <version>2.12.0</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
```

Comment:
Would be nice if we could some how pull in the current spring boot version of this instead of hard coding it here. Other places you don't have to specify the version because the parent pom is spring boot. Makes it easier to fix dependabot issues when we upgrade spring boot versions.

Reply:
fixxed